src/Stock.js

```diff
@@ -94,6 +97,9 @@ class Stock extends React.Component {
                     /> :
                     <h2>Invalid Ticker</h2>
                 }
+                 <div className = "stockInfo">
```

Comment:
Same as other comment. I personally like the extra space between components, so would like one between the plot and the stock info section too if you are okay with that.